composer.json

```diff
@@ -16,10 +16,10 @@
         }
     ],
     "require": {
-        "php" : "^7.0"
+        "php": "^8.0|^8.0"
```

Comment:
Shouldn't we support 7 and 8?

Reply:
Quite correct, completely unintended. I had initially tried to only support PHP 8, then meant to add PHP 7 back in. What I actually did was just copy the version constraint again, instead of adjusting it. Now sorted.